src/api.js

```diff
@@ -54,6 +52,8 @@ const requestPagePreview = ( lang, title, isTouch, callback, request = cachedReq
 				iiextmetadatafilter: 'License|LicenseShortName|ImageDescription|Artist',
 				iiextmetadatalanguage: lang,
 				iiextmetadatamultilang: 1,
+				iiurlwidth: getDeviceSize().width,
+				iiurlheight: getDeviceSize().height,
```

Comment:
instead of having full height, the exact height of the image shown on the screen should be deducted from the minimum image attribution height.

Reply:
Could you elaborate on this point?

Reply:
<img width="785" alt="Screenshot 2020-09-18 113822" src="https://user-images.githubusercontent.com/2560096/93582971-81bf3a00-f9a3-11ea-8669-c476fedb02dc.png">

the red square size is the real width/height for the image

Does this screenshot help to explain?

Reply:
Ohh so you're saying instead of requesting `iiurlheight: getDeviceSize().height` to the api, we do something like `iiurlheight: getDeviceSize().height - 100`? That is, we substract the minimum image attribution height, which is around ~100 px.

Requesting `iiurlheight: getDeviceSize().height - 100` gives us the following

<img width="1537" alt="Screen Shot 2020-09-18 at 9 37 18 AM" src="https://user-images.githubusercontent.com/4752599/93604236-1a00f300-f993-11ea-9d23-e9030507cb3c.png">

So we could do that... but I understood [Sudhanshu's point](https://phabricator.wikimedia.org/T261745#6466762) on using "maximum screen size" as in using full height.

Reply:
you are right, "maximum screen size", let's keep it 100% and don't minus it.